<?xml version="1.0" encoding="UTF-8"?>

<launch> 

	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
	    <param name="serial_name" type="string" value="/dev/ttyAMA0"/>
	    <param name="baud_rate" type="int" value="230400"/>
	    <param name="app_id" type="int" value="1053855"/>
	    <param name="app_version" type="int" value="1"/>
	    <param name="align_time" type="bool" value="false"/>
	    <param name="enc_key" type="string" value="a822aa62212ee4a0f09a2e2b2538e117cf5c6710e4c3e9b675010fb596869087"/>
	    <param name="use_broadcast" type="bool" value="false"/>
    </node>

	<node name="de_airsense_mission" pkg="de_airsense" type="mission_dji_node" output="screen"> </node>

	<node name="de_airsense_waspmote_ipfs" pkg="de_airsense" type="waspmote_ipfs_node" output="screen"> </node>

</launch>
